app/src/main/java/com/prayercompanion/prayercompanionandroid/data/utils/MediaController.kt

```diff
@@ -0,0 +1,28 @@
+package com.prayercompanion.prayercompanionandroid.data.utils
+
+import android.content.Context
+import android.media.AudioAttributes
+import android.media.AudioFocusRequest
+import android.media.AudioManager
+import dagger.hilt.android.qualifiers.ApplicationContext
+import javax.inject.Inject
+
+
+class MediaController @Inject constructor(
+    @ApplicationContext
+    private val context: Context
+){
+    private val audioManager = context.getSystemService(Context.AUDIO_SERVICE) as AudioManager
+
+    fun pauseMedia() {
+        val focusRequest: AudioFocusRequest =
+            AudioFocusRequest.Builder(AudioManager.AUDIOFOCUS_GAIN)
+                .setAudioAttributes(
+                    AudioAttributes.Builder().setUsage(AudioAttributes.USAGE_MEDIA).build()
+                )
+                .build()
```

Comment:
can this be initialized outside of the function so it's not created each time the function is called?

Reply:
resolved in [9d51ebe](https://github.com/Prayer-Companion/PrayerCompanionAndroid/pull/19/commits/9d51ebe66ae0172d935f9658bb85aa752c034953)